### DISCUSSION TITLE
Strange problem: I can't see the jazz category that appear empty

### DISCUSSION THREAD

**Discussion by openaspace** [jamulussoftware/jamulus#3237] [General]
Hello I have a Strange problem: I can't see the jazz category that appear empty!
Tried from 2 differnt mac and same problem.

the other public categories are correctly visibile

**Comment by dcorson-ticino-com** [comment]
Probably network routing problems.
The Any Genre directories are hosted in Germany, the others in England.
Unfortunately we have little to no influence over the network routing.
Do you have the possibility to use another provider?

**Reply by openaspace** [reply]
Thank you.
If I change my IP I can see the directory.

I can know the IP of the jazz server directory? I want to check analytically

**Reply by dcorson-ticino-com** [reply]
Use the Jamulus Explorer, it shows the IP addresses of all the servers.

**Reply by openaspace** [reply]
Changed DNS servers and works correctly
